### ISSUE TITLE
Incorrect version of srclib-javascript toolchan being installed

### ISSUE THREAD

**Issue by alexsaveliev** [sourcegraph/srclib-javascript#31]
- Run `srclib toolchain install javascript`

`git remote -v` gives

```
origin  https://sourcegraph.com/sourcegraph/srclib-javascript (fetch)
origin  https://sourcegraph.com/sourcegraph/srclib-javascript (push)
```

#### Actual result
`git rev-parse HEAD` gives

```
38cb738a5c5b00e89aba23e1d34c166e11460100
```

#### Expected result
`git rev-parse HEAD` gives d18d00fa66ca5c7400c96a289e461199f734ec5a (current HEAD of sourcegraph/srclib-javascript) or later

Please sync sourcegraph.com/sourcegraph/srclib-javascript and github.com/sourcegraph/srclib-javascript

**Comment by beyang** [comment]
I just deleted sourcegraph.com/sourcegraph/srclib-javascript. Let's treat github.com/sourcegraph/srclib-javascript as the canonical location for now.

**Comment by alexsaveliev** [comment]
Closed by sourcegraph/srclib#290
